ui-admin/src/forms/designer/questions/BaseFields.tsx

```diff
@@ -44,24 +48,28 @@ export const BaseFields = (props: BaseFieldsProps) => {
         />
       </div>
 
-      <div className="mb-3">
-        <Textarea
-          infoContent="Optional additional context for the question.
+      <div className="bg-white rounded-3 p-2 mb-2">
+        <CollapsibleSectionButton targetSelector={`#${additionalFieldsTargetId}`}
+          sectionLabel={'Additional options'}/>
+        <div className="mb-3 collapse hide" id={additionalFieldsTargetId}>
+          <Textarea
+            infoContent="Optional additional context for the question.
            Will be displayed in a smaller font beneath the main question text"
-          disabled={disabled}
-          label="Description"
-          rows={2}
-          value={i18nSurveyText(regularQuestion.description)}
-          onChange={value => {
-            onChange({
-              ...regularQuestion,
-              description: value
-            })
-          }}
-        />
+            disabled={disabled}
+            label="Description"
+            rows={2}
+            value={i18nSurveyText(regularQuestion.description)}
+            onChange={value => {
+              onChange({
+                ...regularQuestion,
+                description: value
+              })
+            }}
+          />
+        </div>
```

Comment:
Imo I think it would make sense to move this to the very bottom - it feels like this is a `misc` section that comes after more important things